calrissian/job.py

```diff
@@ -119,6 +119,10 @@ def add_persistent_volume_entries_from_pod(self, pod):
         inspector = KubernetesPodVolumeInspector(pod)
         for mount_path, sub_path, claim_name, read_only in inspector.get_mounted_persistent_volumes():
             self.add_persistent_volume_entry(mount_path, sub_path, claim_name, read_only)
+            # Mount any additional volumes not already mounted, only duplicate is "calrissian-wdir"
+            if claim_name != "calrissian-wdir":
+                log.info("Adding volume mount from workspace")
```

Comment:
Might be useful to log the mount_path and/or claim_name here in case of debug when we start to have many custom volumes

Reply:
Good suggestion, updated logging